bitmap-steven-hawa-caleb/bitmap-transformer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const cliInterpreter = require('./lib/cli-interpreter.js');
+const invert = require('./lib/invert-colors.js');
+const grayscale = require('./lib/grayscale-colors.js');
+const bluescale = require('./lib/bluescale-colors.js');
+const psychedelic = require('./lib/psychedelic.js');
+
+const transformArray = cliInterpreter(process.argv);
+
+// Series of if statements checking for the words "invert," "grayscale," and "bluescale". If they were passed in, use the appropriate module method.
+if (transformArray.some(function(element) { return element.toLowerCase() === 'invert'; })) {
```

Comment:
@maschigokae @BatemanVO @abdih17 great use of the `some` array method in conjunction with your command line arguments